server/entity/workout_logs.go

```diff
@@ -0,0 +1,39 @@
+package entity
+
+import (
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+type WorkoutLog struct {
+	gorm.Model
+	UserID    uint           `gorm:"not null;index"`
+
+	User     User      `gorm:"constraint:OnDelete:CASCADE;foreignKey:UserID"`
+	SetLog  []SetLog  `gorm:"foreignKey:WorkoutLogID;constraint:OnDelete:CASCADE"`
```

Comment:
```suggestion
	SetLogs  []SetLog  `gorm:"foreignKey:WorkoutLogID;constraint:OnDelete:CASCADE"`
```

参照：https://gorm.io/ja_JP/docs/has_many.html